sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -417,6 +417,7 @@ private[hive] object HiveTableUtil {
   def configureJobPropertiesForStorageHandler(
       tableDesc: TableDesc, conf: Configuration, input: Boolean): Unit = {
     val property = tableDesc.getProperties.getProperty(META_TABLE_STORAGE)
+    if (property == null) { return }
```

Comment:
I don't get this, if `property` is null and we pass to `HiveUtils.getStorageHandler`, wouldn't that simply just return `null` and the rest of this method is skipped?